apps/zui/electron-builder.json

```diff
@@ -10,11 +10,15 @@
     "sign": "./scripts/sign.js"
   },
   "linux": {"target": ["deb", "rpm"]},
+  "mac": {
+    "entitlements": "darwin.plist",
+    "notarize": {"teamId": "2DBXHXV7KJ"},
+    "artifactName": "${productName}-${version}-${arch}.${ext}"
```

Comment:
At first when using `electron-builder` at its defaults, we'd get a a DMG for Intel like `Zui-1.7.4.dmg` and for Apple Silicon like `Zui-1.7.4-arm64.dmg`. This use of `artifactName` gives us symmetry by having the Intel one now have the arch included as well and hence `Zui-1.7.4-x64.dmg`.